File: benchmark/fib/fib.js
function fib(n) {
  if (n < 2) {
    return n;
  }
  return fib(n - 1) + fib(n - 2);
}

let total = 0;
for (let rep = 0; rep < 40; rep += 1) {
  total += fib(30);
}
console.log(total);

File: benchmark/fib/fib.lua
local function fib(n)
  if n < 2 then
    return n
  end
  return fib(n - 1) + fib(n - 2)
end

local total = 0
for rep = 1, 40 do
  total = total + fib(30)
end
print(string.format("%d", total))
